postgres/Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG POSTGRES_BASE_IMAGE=postgres:10.12
+FROM $POSTGRES_BASE_IMAGE
+
+# The postgres authentication mode is set to "trust" to avoid the following
+# error:
+#
+#   Error: Database is uninitialized and superuser password is not
+#   specified.  You must specify POSTGRES_PASSWORD for the superuser. Use
+#   "-e POSTGRES_PASSWORD=password" to set it in "docker run".
+#
+#   You may also use POSTGRES_HOST_AUTH_METHOD=trust to allow all
+#   connections without a password. This is *not* recommended. See
+#   PostgreSQL documentation about "trust"
+#
+# This means that either:
+# 1) The Conjur Google Marketplace app (including the backend postgres
+#    pod) should be "siloed" in its own GKE cluster.
+# 2) OR.... a GKE network policy should be configured to limit
+#    ingress and egress in/out of the postgres pod to exclusive access
+#    by the Conjur server.
+ENV POSTGRES_HOST_AUTH_METHOD=trust
```

Comment:
can this change be backed out to separate the postgres changes from the conjur version changes?

Reply:
Yes, I'll remove this.

Reply:
are we leaving this for now? that may be ok if we plan to improve it in a follow-up effort